framework/wazuh/agent.py

```diff
@@ -67,6 +67,47 @@
 ERROR_CODES_UPGRADE_SOCKET_GET_UPGRADE_RESULT = [1813]
 
 
+def build_agents_query(agents_id: list, filters: dict) -> dict:
```

Comment:
I think we are using `agent_list` without the `s` in other places of the framework, so to be consistent we should stick to it. Marking it once but there are other occurrences in the changes

```suggestion
def build_agents_query(agent_list: list, filters: dict) -> dict:
```